main.rb

```diff
@@ -0,0 +1,4 @@
+require_relative "./lib/log_parser.rb"
+
+log_parse = LogParser.new()
```

Comment:
Não precisa user os colchetes:
```suggestion
log_parse = LogParser.new
```

Reply:
Retirei os colchetes.